modules/core/src/main/java/org/apache/ignite/internal/processors/performancestatistics/FilePerformanceStatisticsWriter.java

```diff
@@ -108,6 +113,12 @@
     /** Logger. */
     private final IgniteLogger log;
 
+    /** Hashcodes of cached strings. */
+    private final Set<Integer> cachedStrings = new GridConcurrentHashSet<>();
```

Comment:
Let's rename this to `knownStrs` and `cachedCnt` -> `knownStrCnt`

Reply:
Renamed